src/FsAutoComplete.Core/RecordStubGenerator.fs

```diff
@@ -20,6 +20,7 @@ type RecordExpr = {
     Expr: SynExpr
     CopyExprOption: option<SynExpr * BlockSeparator>
     FieldExprList: SynExprRecordField list
+    LastKnownGoodPosForSymbolLookup: Position
```

Comment:
in case we are at the end of a record declaration (e.g. the `|` in `{ a = "";| }`), we need _some_ kind of identifier to use to look up the symbol to find the record's type definition. this field allows us to grab that during the AST walk for later use.